test/utils/create-libp2p-node.js

```diff
@@ -8,33 +8,46 @@ const KadDHT = require('libp2p-kad-dht')
 const waterfall = require('async/waterfall')
 const PeerInfo = require('peer-info')
 const PeerId = require('peer-id')
+const defaultsDeep = require('@nodeutils/defaults-deep')
 
 class Node extends libp2p {
-  constructor (peerInfo, options) {
-    options = options || {}
-
-    const modules = {
-      transport: [new TCP()],
-      connection: {
-        muxer: Multiplex,
-        crypto: SECIO
+  constructor (_options) {
```

Comment:
Why do you use an underscore in front of `options`. I normally use a leading underscore to signal that a variable isn't used (like Erlang and Rust do it).